### PR TITLE
changes for popus in four pages

### PR DESCRIPTION
**Pull request by vivektripathi2020**
changes for popus in four pages

### CODE REVIEW

File: source/_patterns/05-pages/my-account-label-page.yml [hunk 1]
```diff
@@ -203,4 +205,78 @@ myaccount_bottom:
...
+      info: Close
+      togglemodal: modal
+      type: button
+      style: secondary-light 
+      text: Cancel
+    saveButton:
+      type: button
+      style: primary 
+      text: Save
+myaccountalert_delete:
```
Comment (vinothk-hw) [inline source/_patterns/05-pages/my-account-label-page.yml:233]: use generic object name for all delete popup.

File: source/_patterns/03-components/modal/modal-popup.twig [hunk 2]
```diff
@@ -38,15 +38,50 @@
...
+					{% for item in modalPopup.myaccount_alert_addlabel %}
+						<div class="my-account__alertpopup">	
+							{% include "@atoms/forms/inpu-text.twig" with {
+								field: item.modalInput
+							} only %}			
 						</div>
 					{% endfor %}
 				{% endif %}
+				{% if modalPopup.myaccountlabel %}
+                    {% for item in modalPopup.myaccountlabel %}
```
Comment (vinothk-hw) [inline source/_patterns/03-components/modal/modal-popup.twig:78]: add and edit object should be proper

File: source/_patterns/03-components/modal/modal-popup.twig [hunk 3]
```diff
@@ -38,15 +38,50 @@
...
+					<div class="my-account__alert-radio-box">				
+						{% for item in modalPopup.modalradiobox %}						
+							{% include "@atoms/forms/radiobox.twig" with {
+								radiobox: item.modalradio
+							} only %}						
+						{% endfor %}
+					</div>							
+				{% endif %}
+				{% if modalPopup.myaccount_alert_delete %}
+					{% for item in modalPopup.myaccount_alert_delete %}
```
Comment (vinothk-hw) [inline source/_patterns/03-components/modal/modal-popup.twig:56]: delete object name should be generic

File: source/_patterns/03-components/site/my-account-alerts.yml [hunk 4]
```diff
@@ -179,4 +181,30 @@ myaccount_bottom:
...
+      text: Delete  
+    myaccount_alert_delete:
+    - alertheading:
+        level: '3'
+        text: Are you sure?
+        class: text-center
+      alertdesc:
+        class: text-center
+        text: Do you really want to delete these records? This process cannot be undone  
+    closeButton:
```
Comment (vinothk-hw) [inline source/_patterns/03-components/site/my-account-alerts.yml:201]: Refer Search result modal popup use same style of save and cancel

**Review by vinothk-hw — COMMENTED** [review]
review comments and implement the same...

File: source/_patterns/03-components/site/my-account-saved-search.yml [hunk 5]
```diff
@@ -127,4 +139,30 @@ myaccount_bottom:
...
+      title: '100'
+myaccountalert_delete:
+  modalPopup:
+    modaltargetid: account_alert_delete
+    modal_type: modal-small-tablet
+    modalHeading:
+      level: 6
+      class: modal-title
+      text: Delete  
+    myaccount_alert_delete:
```
[the remark this answers is not in the document]
Comment (vivektripathi2020) [reply source/_patterns/03-components/site/my-account-saved-search.yml]: renamed object removed alert

File: source/_patterns/03-components/site/my-account-saved-search.twig [hunk 6]
```diff
@@ -57,6 +57,9 @@
     {% include "@components/modal/modal-popup.twig" with {
 		modalPopup: myaccountsavedsearch_edit.modalPopup
 	} only %}
+    {% include "@components/modal/modal-popup.twig" with {
```
[the remark this answers is not in the document]
Comment (vivektripathi2020) [reply source/_patterns/03-components/site/my-account-saved-search.twig:60]: renamed object 
added spacing in saved search popup
and added close button functionality

File: source/_patterns/05-pages/my-account-favouite-page.yml [hunk 7]
```diff
@@ -218,4 +218,30 @@ myaccount_bottom:
...
+      title: '100'
+myaccountalert_delete:
+  modalPopup:
+    modaltargetid: account_alert_delete
+    modal_type: modal-small-tablet
+    modalHeading:
+      level: 6
+      class: modal-title
+      text: Delete  
+    myaccount_alert_delete:
```
[the remark this answers is not in the document]
Comment (vivektripathi2020) [reply source/_patterns/05-pages/my-account-favouite-page.yml]: delete popup object renamed to common

File: source/_patterns/05-pages/my-account-label-page.yml (hunk 1 above)
Comment (vivektripathi2020) [reply source/_patterns/05-pages/my-account-label-page.yml]: delete popup object renamed to common

File: source/_patterns/03-components/modal/modal-popup.twig (hunk 2 above)
Comment (vivektripathi2020) [reply source/_patterns/03-components/modal/modal-popup.twig]: for edit popup name changed like add

File: source/_patterns/03-components/site/my-account-alerts.yml (hunk 4 above)
Comment (vivektripathi2020) [reply source/_patterns/03-components/site/my-account-alerts.yml:201]: applied same style

**Review by vinothk-hw — APPROVED** [review]

**Pull request merged**